examples/decomposition/plot_image_denoising.py

```diff
@@ -31,9 +31,10 @@
 necessarily related to visualisation.
 
 """
+from time import time
+
```

Comment:
please remove this and the import

Reply:
sorry, import shouldn't be removed, but please leave it where it was to avoid unnecessary changes

Reply:
ok, I will do it.